Sources_v3/Workers/Background/WebSocketConnectedProcessor.swift

```diff
@@ -0,0 +1,65 @@
+//
+// Copyright © 2020 Stream.io Inc. All rights reserved.
+//
+
+import CoreData
+
+/// After WebSocket changes it's state to `connected` we need to start watching existing channels so we can receive updated on them.
+/// This background worker observes the WebSocket state and on `connected` we are fetching all channels from DB and sending
```

Comment:
nit: ~observes the WebSocket state~ -> listents to `ConnectionStatusUpdated` event ?